main.go

```diff
@@ -141,6 +141,15 @@ func handleRepo(ctx context.Context, client *github.Client, repo *github.Reposit
 	opt := &github.ListOptions{
 		PerPage: 100,
 	}
+
+	teams, resp, err := client.Repositories.ListTeams(ctx, *repo.Owner.Login, *repo.Name, opt)
+	if resp.StatusCode == http.StatusNotFound || resp.StatusCode == http.StatusForbidden {
+		return nil
```

Comment:
I wonder if we should return an Error here instead.

Reply:
Same as above, I have just reused the same error handler code which is implemented on all other cases. Anyway, if you take a look to next lines, you can see the error object `err` is returned only if it is not null. Maybe the original author wants to interrupt execution only if `err` is not null?

Reply:
Yeah, good thought. How about leaving it as-is, and asking the author when you file upstream?

Reply:
:+1: